lib/Service/SendMailService.php

```diff
@@ -88,6 +88,93 @@ public function __construct(
 	}
 
 
+	/**
+	 * @param string $author
+	 * @param Circle $circle
+	 * @param Member $member
+	 * @param array $mails
+	 */
+	public function generateInvitationMail(
+		string $author,
+		Circle $circle,
+		Member $member,
+		array $mails,
+	): void {
+
+
+		if ($member->getUserType() === Member::TYPE_MAIL) {
+			$mails = [$member->getUserId()];
+		}
+
+		if (empty($mails)) {
+			return;
+		}
+
+		$circleName = $circle->getDisplayName();
+
+		$template = $this->generateMailInvitation(
+			$author,
+			$circleName
+		);
+
+		foreach ($mails as $mail) {
+			try {
+				$this->sendMailInvitation($template, $author, $mail, $circleName);
+			} catch (Exception $e) {
+			}
+		}
```

Comment:
What happens during this exception? You probably want to write this on the logs?